nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -496,6 +496,14 @@ Use the following commands:
     Retype new password: ***
     ```
 
+    If you have a user account declared in your `configuration.nix`
+    and plan to log in using this user, you should set a password
+    before rebooting, e.g. for the `alice` user:
+
+    ```ShellSession
+    # nixos-enter --root /mnt -c 'passwd alice'
+    ```
```

Comment:
The instruction is already conditional, so I don't think it's "should" but "must", at which point we can use imperative.

````suggestion
    If you have a user account declared in your `configuration.nix` and plan to log in using this user, set a password before rebooting, e.g. for the `alice` user:

    ```ShellSession
    # nixos-enter --root /mnt -c 'passwd alice'
    ```
````